conformance/test/server.py

```diff
@@ -619,14 +657,14 @@ def _find_free_port():
 
 class Args(argparse.Namespace):
     mode: Literal["sync", "async"]
-    server: Literal["granian", "hypercorn", "uvicorn"]
+    server: Literal["daphne", "granian", "hypercorn", "uvicorn"]
```

Comment:
nit: should this also have "gunicorn" in the list?

Reply:
Yeah I guess there's something causing pyright to not flag this